README.rst

```diff
@@ -158,50 +158,68 @@ be used are:
+      -cw WORKERS, --celery-workers WORKERS
+                            Celery number of workers
+                            env: MICRO_CELERY_WORKERS
+                            default: 1
+      -G, --gunicorn        plugins available through API Rest
+                            env: MICRO_GUNICORN
+                            default: None
+      -gw WORKERS, --gunicorn-workers WORKERS
+                            Gunicorn number of workers
+                            env: MICRO_GUNICORN_WORKERS
+                            default: 1
       -H HOSTNAME, --hostname HOSTNAME
                             Celery worker's hostname
-      -w NUM_WORKERS, --num-workers NUM_WORKERS
-                            set the Celery worker number
-      -bi BIND, --bind BIND
-                            Set the Gunicorn socket bind, HOST:PORT
+                            env: MICRO_HOSTNAME
+                            default: micro
+      -ln LOG_FILE_NAME, --log-file-name LOG_FILE_NAME
+                            Micro's log file name
+                            env: MICRO_LOG_FILE_NAME
+                            default: micro.log
+      -lp LOG_FOLDER_PATH, --log-folder-path LOG_FOLDER_PATH
+                            path to the Micro's log folder
+                            env: MICRO_LOG_FOLDER_PATH
+                            default: /var/log/micro
       -ll LOG_LEVEL, --log-level LOG_LEVEL
-                            log level: DEBUG, INFO, WARNING, ERROR, CRITICAL or
-                            FATAL
-      -lp LOG_PATH, --log-path LOG_PATH
-                            log file path
-      -cll CELERY_LOG_LEVEL, --celery-log-level CELERY_LOG_LEVEL
-                            Celery log level: DEBUG, INFO, WARNING, ERROR,
-                            CRITICAL or FATAL
-      -clp CELERY_LOG_PATH, --celery-log-path CELERY_LOG_PATH
-                            Celery log file path
-      -cpp CELERY_PID_PATH, --celery-pid-path CELERY_PID_PATH
-                            Celery PIDs path
-      --default-params      show default parameters
-      --version             show Micro version
-      -h, --help            Show this help message
+                            Micro's log level
+                            env: MICRO_LOG_LEVEL
+                            default: WARNING
+      -pp PID_FOLDER_PATH, --pid-folder-path PID_FOLDER_PATH
+                            path to the Micro's PID folder
+                            env: MICRO_PID_FOLDER_PATH
+                            default: /var/run/micro
+      -p PLUGIN_PATH, --plugin-path PLUGIN_PATH
+                            path to the plugins folder
+                            env: MICRO_PLUGIN_PATH
+                            default: None
+      -q TASK_QUEUES, --task-queues TASK_QUEUES
+                            Celery task queues
+                            env: MICRO_TASK_QUEUES
+                            default: None
 
 Environment variables
```

Comment:
```suggestion
**IMPORTANT:** `-G`, `--gunicorn`  or `-C`, `--celery`, or both, must be included otherwise Micro will finish without any message
 
Environment variables
```